Update roxterm.desktop (pt_BR by marcelocripe and fr_BE by Wallon)

marcelocripe opened the pull request:
Update roxterm.desktop:
pt_BR by marcelocripe and fr_BE by Wallon

In the "Traducao do ROXTerm 02-05-2024.zip" there are the ".po" and ".mo" files with the revision and improvements of the translations of the "pt_BR" language.

Thank you very much for programming, developing and keeping ROXTerm alive.

marcelocripe

[Traducao do ROXTerm 02-05-2024.zip](https://github.com/realh/roxterm/files/15191512/Traducao.do.ROXTerm.02-05-2024.zip)

marcelocripe commented:
Please add the files from the folders https://sourceforge.net/p/roxterm/code/ci/master/tree/po/ to the GitHub project
and which are not in https://sourceforge.net/p/roxterm/code/ci/master/tree/po4a/ which are not in https://github.com/realh/roxterm/.

The compressed file "po and po4a.zip" has the files that are on SourceForge that are not on GitHub. I included in the "po" folder the "pt_BR.po" file that I reviewed.

Thank you very much for programming, developing and keeping ROXTerm alive.

[po and po4a.zip](https://github.com/realh/roxterm/files/15295371/po.and.po4a.zip)

realh commented:
I've added the contents of the zip, and made sure POTFILES.in is up-to-date, but not integrated it into the build system because CMake is unusable.

marcelocripe commented:
> I've added the contents of the zip, and made sure POTFILES.in is up-to-date, but not integrated it into the
> build system because CMake is unusable.

Bringing SourceForce files to this repository is already very important for the project and will facilitate the voluntary work of ROXTerm packagers.

Despite having the note "Restored (very) old translation files" in the "pt_BR.po" file, I updated it in May 2024.

We in the antiX community sent the ROXTerm ".pot" file to the translation platform on July 23, 2024 https://app.transifex.com/antix-linux-community-contributions/antix-contribs/roxterm/.

I would like to know if I can send you in the same merge request the ".po" files from several other languages ​​when they are translated by the other volunteer translators?

Thank you very much.

- - - - -

> I've added the contents of the zip, and made sure POTFILES.in is up-to-date, but not integrated it into the
> build system because CMake is unusable.

Trazer os arquivos do SourceForce para este repositório já é algo muito importante para o projeto e facilitará o trabalho voluntário dos empacotadores do ROXTerm.

Apesar de ter a observação "Restored (very) old translation files" no arquivo "pt_BR.po" eu o atualizei em maio de 2024.

Nós da comunidade do antiX, enviamos no dia 23 de julho de 2024 o arquivo ".pot" do ROXTerm para a plataforma de tradução https://app.transifex.com/antix-linux-community-contributions/antix-contribs/roxterm/.

Eu gostaria de saber se eu posso te enviar neste mesmo pedido de mesclagem os arquivos ".po" de vários outros idiomas quando eles forem traduzidos pelos outros tradutores voluntários?

Muito obrigado.
